tests/contrib/aiopg/py35/test.py

```diff
@@ -52,7 +52,7 @@ async def _test_cursor_ctx_manager(self):
             assert len(rows) == 1
             assert rows[0][0] == 'blah'
 
-        spans = tracer.writer.pop()
+        spans = tracer.pop()
```

Comment:
Question: why do we use `tracer.pop()` here compared to `self.pop_spans()`? Is there a difference between using the tracer's method to pop or the TestCase method to pop?

Reply:
Good catch! This one can be `self.pop_spans()`. I had missed converting this one over. There really isn't a difference but by using the unittest style one we get consistency and avoid using the not obvious `tracer.pop()` proxy.